### PR TITLE
Copter: Correct the typo

### PR DESCRIPTION
**Pull request by muramura**
I would like to make the following changes:

1. The Gregorian calendar year is set in the future.
2. There is no beta1 release.

### CODE REVIEW

File: ArduCopter/ReleaseNotes.txt [hunk 1]
```diff
@@ -989,7 +989,7 @@ Changes from 4.3.6
     e) Memory corruption bug in the STM32H757 (very rare)
     f) RC input on IOMCU bug fix (RC might not be regained if lost)
 ------------------------------------------------------------------
-Copter 4.3.6 05-Apr-2023 / 4.3.6-beta1, 4.3.6--beta2 27-Mar-2023
```
Comment (rmackay9) [inline ArduCopter/ReleaseNotes.txt:992]: Hi @muramura,

Thanks for this, I think 4.3.6-beta2 was a real release but it only lasted for a few hours.  Could you change "4.3.6--beta1"
 to "4.3.6-beta2"?  After that I think we can merge this.

**Comment by muramura** [comment]
I would like to graph the releases, the number of bug fixes, and the approximate amount of code from AC3.6.11 to AC4.5.7, in order to understand the trend of future bug fix numbers.

In this investigation, I focused on the number of bug fix items in the release notes.


4dc386e6-a004-4fd8-8e33-f323b73595f4

**Comment by muramura** [comment]
I would like to graph the releases, the number of bug fixes, and the approximate amount of code from AC3.6.11 to AC4.5.7, in order to understand the trend of future bug fix numbers.

In this investigation, I focused on the number of bug fix items in the release notes.

File: ArduCopter/ReleaseNotes.txt (hunk 1 above)
Comment (muramura) [reply ArduCopter/ReleaseNotes.txt:992]: So beta2 has been around for a few hours, hasn't it? I have fixed it.

Comment (rmackay9) [reply ArduCopter/ReleaseNotes.txt:992]: Hi @muramura,

Sorry to be nitpicky but -beta2 was released at the same time as beta1.  The changes made above make it appear like it was released at the same time as the stable.  I think the line should look like this:

`Copter 4.3.6 05-Apr-2023 / 4.3.6-beta1, 4.3.6-beta1 27-Mar-2023`

**Comment by muramura** [comment]
@rmackay9 san. 

I agreed with the comment. I have made the changes.

File: ArduCopter/ReleaseNotes.txt (hunk 1 above)
Comment (muramura) [reply ArduCopter/ReleaseNotes.txt:992]: I agreed with the comment. I have made the changes.

Comment (rmackay9) [reply ArduCopter/ReleaseNotes.txt:992]: Sorry it should have been, 

Copter 4.3.6 05-Apr-2023 / 4.3.6-beta1, 4.3.6-beta2 27-Mar-2023

**Review by rmackay9 — APPROVED** [review]

**Pull request merged**